### PR TITLE
Remove Juju install mentions for appliance

### PR DESCRIPTION
**Pull request by keirthana**

### CODE REVIEW

File: ref/requirements.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Anbox Cloud currently supports the following LXD versions:
 <a name="juju-version"></a>
 ### Juju version
 
-Anbox Cloud requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. Starting with Anbox Cloud 1.13, the required Juju version is **2.9**. Earlier versions require Juju **2.8**.
+The regular Anbox Cloud variant requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. Starting with Anbox Cloud 1.13, the required Juju version is **2.9**. Earlier versions require Juju **2.8**.
```
Comment (morphis) [inline ref/requirements.md:80]:
```suggestion
The regular Anbox Cloud variant requires [Juju](https://juju.is/) to be installed to manage the different components and their dependencies. The minimum required Juju version is **2.9**.
```

Lets drop the mention of 2.8 entirely. We support/can work with 2.9+. CI actually tests 3.1 today by default.

**Review by morphis — CHANGES_REQUESTED** [review]

File: tut/installing-appliance.md [hunk 2]
```diff
@@ -66,12 +66,7 @@ Run the following command to install the `anbox-cloud-appliance` snap, which han
 <a name="additional-tools"></a>
 ### 5. Install additional tools
```
Comment (ajanon) [inline tut/installing-appliance.md:67]: Should we keep it in its own subsection or should we merge two subsections?

This may be better left to another PR if you want to keep it small in scope, though.

**Review by ajanon — APPROVED** [review]
LGTM, one comment

File: tut/installing-appliance.md (hunk 2 above)
Comment (keirthana) [reply tut/installing-appliance.md:67]: No, I think it can stay in a separate section because it is a tutorial and it is useful for readers to understand that AMC is an additional tool we are installing on top of the appliance. Also, the additional tools sub section is referred in the getting started tutorial as a prereq.

**Review by morphis — APPROVED** [review]
LGTM
